backend/src/main/java/co/inventorsoft/scripty/service/SecurityService.java

```diff
@@ -6,17 +6,35 @@
 import org.springframework.stereotype.Service;
 
 import co.inventorsoft.scripty.exception.ApplicationException;
+import co.inventorsoft.scripty.model.entity.Project;
+import co.inventorsoft.scripty.repository.ProjectRepository;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
 
 /**
  * @author lzabidovsky 
  */
 @Service
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
 public class SecurityService {
-	
+
+	ProjectRepository projectRepository;
+
 	public void authenticationHasRoleUser(Authentication authentication) {
 		if(!authentication.getAuthorities().contains(new SimpleGrantedAuthority("ROLE_USER"))) {
 			throw new ApplicationException(authentication.getName()+" does not have ROLE_USER" , HttpStatus.FORBIDDEN);
 		}
 	}
 
+	public Project projectUserIsOwner(Long projectId, Authentication auth) {
```

Comment:
This service should check permissions - nothing more. Please, pass all required entities(like project in this case) as an argument to method and load project somewhere else(ProjectService is the natural fit)